build/eslint/xfail.list

```diff
@@ -1,11 +1,237 @@
+apps/system/js/card.js
+apps/system/js/entry_sheet.js
+apps/system/js/global_overlay_window.js
+apps/system/js/ime_menu.js
+apps/system/js/permission_manager.js
+apps/system/js/roaming_icon.js
+apps/system/js/signal_icon.js
+apps/system/js/system_dialog.js
+apps/system/js/time_icon.js
+apps/system/js/ttl_view.js
+apps/system/js/value_selector/value_selector.js
+apps/system/lockscreen/js/widgets/clock/lockscreen_clock_widget.js
+apps/system/mobile_id/js/error_overlay.js
+apps/system/mobile_id/js/ui.js
+apps/system/test/unit/app_install_manager_test.js
+apps/system/test/unit/app_modal_dialog_test.js
+apps/system/test/unit/battery_overlay_test.js
+apps/system/test/unit/crash_reporter_test.js
+apps/system/test/unit/device_storage_watcher_test.js
+apps/system/test/unit/system_banner_test.js
+apps/system/test/unit/text_selection_dialog_test.js
+apps/system/test/unit/update_manager_test.js
+apps/system/test/unit/value_picker_test.js
+apps/system/test/unit/value_selector_test.js
+apps/video/js/thumbnail_date_group.js
+apps/video/js/thumbnail_item.js
+apps/video/test/unit/mock_thumbnail_group.js
+build/test/unit/webapp_optimize_test.js
+build/utils-node.js
 build/utils-xpc.js
+build/webapp-optimize.js
+shared/elements/gaia-component/gaia-component.js
+shared/elements/gaia_grid/script.js
+shared/elements/gaia-header/dist/gaia-header.js
+shared/elements/gaia_radio/script.js
+shared/elements/gaia_sim_picker/script.js
+shared/elements/gaia_textinput/script.js
+shared/elements/gaia-theme/lib/gaia-theme-selector.js
+shared/js/component_utils.js
+shared/js/contacts/search.js
+shared/js/contacts/utilities/templates.js
+shared/js/homescreens/confirm_dialog_helper.js
+shared/js/html_imports.js
+shared/js/l10n.js
+shared/js/lazy_loader.js
+shared/js/smart-screen/shared_utils.js
+shared/js/template.js
+shared/js/utilities.js
+shared/test/unit/load_body_html_helper.js
 tests/atoms/remote_date.js
 tests/atoms/screenshot.js
+tests/jsmarionette/client/marionette-client/docs/theme/assets/vendor/prettify/prettify-min.js
 tests/python/gaia-ui-tests/gaiatest/atoms/remote_date.js
 tests/python/gaia-ui-tests/gaiatest/atoms/screenshot.js
+tv_apps/fling-player/bower_components/gaia-component/gaia-component.js
+tv_apps/fling-player/bower_components/gaia-theme/lib/gaia-theme-selector.js
+tv_apps/smart-home/js/home.js
+tv_apps/smart-system/bower_components/gaia-component/gaia-component.js
+tv_apps/smart-system/js/activity_window.js
+tv_apps/smart-system/js/app_chrome.js
+tv_apps/smart-system/js/app_install_manager.js
+tv_apps/smart-system/js/attention_indicator.js
+tv_apps/smart-system/js/attention_window.js
+tv_apps/smart-system/js/base_ui.js
+tv_apps/smart-system/js/entry_sheet.js
+tv_apps/smart-system/js/ime_menu.js
+tv_apps/smart-system/js/modal_dialog.js
+tv_apps/smart-system/js/permission_manager.js
+tv_apps/smart-system/js/ttlview.js
+tv_apps/smart-system/js/value_selector/value_selector.js
+tv_apps/weather-widget/web_components/smart-weather.js
```

Comment:
that's what I was saying in the previous patch: we need a way to say "this file is in the xfail.list because it fails this test"; at least a comment we could add manually, or even better: it could be checked by the checking script.